### DISCUSSION TITLE
Returning notFound in getServerSideProps causes an error.

### DISCUSSION THREAD

**Discussion by jaeheeko** [vercel/next.js#19772] [Help] [answered]
Hi. I'm using Next.js 9.5.5 and love to develop with it so far.

When I try to return { notFound: true } in getServerSideProps, it causes an error like below.
![image](https://user-images.githubusercontent.com/712762/100986510-a1243700-3590-11eb-9386-8deb67299eaf.png)

But the [API reference (9.3 and up)](https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering) says it's possible to return `notFound` or `redirect` as a return object. 
![image](https://user-images.githubusercontent.com/712762/100986380-76d27980-3590-11eb-8c83-ed3de35a9a25.png)

Did I miss something?

**Comment by jamesmosier** [comment, marked as answer]
It looks like you should only return `return { notFound: true }` and no other props. Is that way your `return` statement looks like?

**Reply by jaeheeko** [reply]
Yes, my code just looks like this.

```
if (!playable) {
    return {
        notFound: true,
    };
}
```

**Reply by jaeheeko** [reply]
I've just updated Next.js version to 10.0.3 and now it works. Maybe the supported version specified in the documentation is mismatched?

**Reply by jamesmosier** [reply]
9.3 is for the addition of getStaticProps, getStaticPaths, and getServerSideProps

`notFound` was more recent: https://github.com/vercel/next.js/pull/18241

**Comment by ByBogon** [comment]
I also got that error message with 11.1.2 version.
<img width="143" alt="스크린샷 2021-11-08 오후 12 14 35" src="https://user-images.githubusercontent.com/18511700/140678725-a7ebe697-b630-4f43-937c-1e0a0e8af7ac.png">

**Reply by ByBogon** [reply]
NVM. It was my fault that I typed notfound, not notFound.
